### PR TITLE
Add Lars

### LINKED ISSUE #2337: Adding Lars Optimizer to TF addons
**Describe the feature and the current behavior/state.**

**Relevant information**
- Are you willing to contribute it (yes/no):
- Yes
- Are you willing to maintain it going forward? (yes/no):
- Yes
- Is there a relevant academic paper? (if so, where):
- Yes, https://arxiv.org/abs/1708.03888 (It also crosses the 100 citations threshold as discussed in #2194 )
- Is there already an implementation in another framework? (if so, where):
- Was it part of tf.contrib? (if so, where):
- Yes,  https://github.com/tensorflow/tensorflow/blob/r1.15/tensorflow/contrib/opt/python/training/lars_optimizer.py

**Which API type would this fall under (layer, metric, optimizer, etc.)**
Optimizer

CC: @bhack

### PR DESCRIPTION
**Pull request by fsx950223**
# Description

Brief Description of the PR:
Add LARS optimizer
Fixes # (issue)
Fix #2337
## Type of change

- [ ] Bug fix
- [ ] New Tutorial
- [ ] Updated or additional documentation
- [ ] Additional Testing
- [ ] New Activation and the changes conform to the [activation contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/activations/README.md#contribution-guidelines)
- [ ] New Callback and the changes conform to the [callback contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/callbacks/README.md#contribution-guidelines)
- [ ] New Image addition and the changes conform to the [image op contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/image/README.md#contribution-guidelines)
- [ ] New Layer and the changes conform to the [layer contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/layers/README.md#contribution-guidelines)
- [ ] New Loss and the changes conform to the [loss contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/losses/README.md#contribution-guidelines)
- [ ] New Metric and the changes conform to the [metric contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/metrics/README.md#contribution-guidelines)
- [x] New Optimizer and the changes conform to the [optimizer contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/optimizers/README.md#contribution-guidelines)
- [ ] New RNN Cell and the changes conform to the [rnn contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/rnn/README.md#contribution-guidelines)
- [ ] New Seq2seq addition and the changes conform to the [seq2seq contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/seq2seq/README.md#contribution-guidelines)
- [ ] New Text addition and the changes conform to the [text op contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/text/README.md#contribution-guidelines)

# Checklist:

- [x] I've properly [formatted my code according to the guidelines](https://github.com/tensorflow/addons/blob/master/CONTRIBUTING.md#coding-style)
    - [x] By running Black + Flake8
    - [x] By running pre-commit hooks
- [x] This PR addresses an already submitted issue for TensorFlow Addons
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] This PR contains modifications to C++ custom-ops

# How Has This Been Tested?

If you're adding a bugfix or new feature please describe the tests that you ran to verify your changes:
*

### CODE REVIEW

**Comment by bhack** [comment]
We are going to duplicate ecosystem LARS https://github.com/tensorflow/models/blob/master/official/modeling/optimization/lars_optimizer.py#L25 /cc @tomerk
